### PR TITLE
KER-203: Hide creator email from hearing report

### PR DESCRIPTION
**Pull request by danipran**

### CODE REVIEW

File: democracy/views/hearing_report.py [hunk 1]
```diff
@@ -202,13 +201,11 @@ def add_comment_row(self, comment, section_worksheet):
             name = comment.get('creator_name')
```
Comment (voneiden) [inline democracy/views/hearing_report.py:201]:
```suggestion
            name = comment.get('author_name')
```

File: democracy/views/section_comment.py [hunk 2]
```diff
@@ -251,9 +251,11 @@ def get_can_delete(self, comment: SectionComment):
     def to_representation(self, instance):
```
Comment (voneiden) [inline democracy/views/section_comment.py:251]: No need to show these fields to anyone (according to smoi) unless FE needs them. Checked that FE does not need them. So could could just use `settings.HEARING_REPORT_PUBLIC_AUTHOR_NAMES` to determine whether to include `creator_email` in the serializer or not.

File: democracy/tests/integrationtest/test_comment.py [hunk 3]
```diff
@@ -796,50 +796,65 @@ def test_56_get_hearing_with_section_check_n_comments_property(api_client, get_c
...
     section = default_hearing.sections.first()
-    url = get_hearing_detail_url(default_hearing.id, 'sections/%s/comments' % section.id)
-    response = john_doe_api_client.get(url)
-    data = get_data_from_response(response, 200)
+
+    data = get_section_comments(default_hearing, section, john_doe_api_client)
     # check no section comment has creator_name when not authorized and the setting is disabled
     for comment in data:
         assert not "creator_name" in comment
+        assert not "creator_email" in comment
```
Comment (voneiden) [inline democracy/tests/integrationtest/test_comment.py:813]:
```suggestion
        assert "creator_email" not in comment
```

File: democracy/tests/integrationtest/test_comment.py [hunk 4]
```diff
@@ -796,50 +796,65 @@ def test_56_get_hearing_with_section_check_n_comments_property(api_client, get_c
...
     section = default_hearing.sections.first()
-    url = get_hearing_detail_url(default_hearing.id, 'sections/%s/comments' % section.id)
-    response = john_doe_api_client.get(url)
-    data = get_data_from_response(response, 200)
+
+    data = get_section_comments(default_hearing, section, john_doe_api_client)
     # check section comment has creator_name when not authorized and the setting is enabled
     for comment in data:
         assert "creator_name" in comment
+        assert "creater_email" not in comment
```
Comment (voneiden) [inline democracy/tests/integrationtest/test_comment.py:825]:
```suggestion
        assert "creator_email" not in comment
```

**Review by voneiden — CHANGES_REQUESTED** [review]

File: democracy/tests/integrationtest/test_comment.py (hunk 3 above)
Comment (danipran) [reply democracy/tests/integrationtest/test_comment.py:813]: 🤡

**Review by voneiden — APPROVED** [review]
Seems good.

**Pull request merged**